index.html

```diff
@@ -0,0 +1,282 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Tesfaye-Portifolio-Project</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <div>
+
+    <nav>
+      <h3 class="logo pointer">TESFAYE.T</h3>
+      <div>
+        <ul id="desk-list">
+          <li class="pointer">Portoflio</li>
+          <li class="pointer">About</li>
+          <li class="pointer">Contact</li>
+        </ul>
+        <img class="pointer" id="Union-menu" src="./images/Union.svg" alt="Union" />
+      </div>
+    </nav>
+
+    <main>
+      <section>
+        <h1>I'm Tesfaye <br>Glad to see you!</h1>
+        <p>
+          I'm a software developer! I can help you build a product , feature
+          or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+
+        <a class="pointer btn-action">LET'S CONNECT</a>
+
+        <ul class="socials">
+          <li class="pointer">
+            <img src="./images/twitter.svg" alt="Twitter" />
+          </li>
+          <li class="pointer">
+            <img src="./images/Linkedin.svg" alt="Linkedin" />
+          </li>
+          <li class="pointer">
+            <img src="./images/medium.svg" alt="Medium" />
+          </li>
+          <li class="pointer">
+            <img src="./images/github.svg" alt="Github" />
+          </li>
+          <li class="pointer">
+            <img src="./images/angellist.svg" alt="Angellist" />
+          </li>
+        </ul>
+      </section>
+    </main>
+
+    <section id="projects">
+      <div class="card">
+        <div class="card-img">
+          <img src="./images/Snapshoot-Portfolio-4.png" alt="Tonic" />
+        </div>
+
+        <div class="details">
+          <h2>Multi-Post Stories</h2>
+          <ul>
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <menu>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </menu>
+          <a class="btn-action">See Project</a>
+        </div>
```

Comment:
- [ ] Please reduce the left-right margin from the project card and make sure it matches [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) the design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/187044668-ac8733dd-84fb-4738-a884-41145a73ed22.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/187044676-0b35081f-f6aa-4c95-8ea7-95d228bb844b.png)
****